org.gnome.Shotwell.json

```diff
@@ -150,15 +150,39 @@
                 }
             ]
         },
+        {
+            "name": "libportal",
+            "buildsystem": "meson",
+            "builddir": true,
+            "config-opts": [
+                "--libdir=/app/lib",
+                "--buildtype=debugoptimized",
+                "-Dbackends=gtk3",
+                "-Ddocs=false",
+                "-Dtests=false"
+            ],
+            "sources" : [
+                {
+                    "type": "git",
+                    "url": "https://github.com/flatpak/libportal.git",
+                    "branch": "main",
```

Comment:
never put a branch. this will break on new commits.

Reply:
there is a commit hash as well

Reply:
Or do you mean to just use commit without branch?

Reply:
so when there is a new commit added branch and commit won't match.

This is exactly what I'm saying.

I have tried to make this as an error with the linter and got overruled. (it's still a warning).

Reply:
I wrote "never put a branch.". Not sure how ambiguous that is.

Reply:
To my cold-ridden slighty too tired brain it was. but you are right, it is pretty obvious